docs/google-cloud.md

```diff
@@ -10,7 +10,7 @@ The purpose of this walkthrough is to guide you through the configuration of the
 
   **Note**: If not already doing so, you can also open this walkthrough directly in [Google Cloud Shell](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/google/kctf&tutorial=docs/google-cloud.md&shellonly=true).
```

Comment:
can we add a note here that you should install netcat in that case and replace all mentions of telnet with netcat below?

Reply:
is there a downside to telnet?

Reply:
yes, it has weird behavior with newlines. I think it sends \r?
Also if I see a project talking about telnet I will assume it's from the 90s ;)

Reply:
ok, yea makes sense.